src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -196,8 +196,23 @@ public Path getJavaAppEngineSdkPath() {
     return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
   }
 
-  private Path getWindowsPythonPath() {
-    return getSdkPath().resolve(WINDOWS_BUNDLED_PYTHON);
+  // https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/189
+  @VisibleForTesting
+  Path getWindowsPythonPath() {
+    String cloudSdkPython = System.getenv("CLOUDSDK_PYTHON");
+    if (cloudSdkPython != null) {
+      Path cloudSdkPythonPath = Paths.get(cloudSdkPython);
+      if (Files.isExecutable(cloudSdkPythonPath)) {
```

Comment:
I think I would skip the isExecutable check. If the user tried to specify the wrong path, let it crash so they can fix their mistake.